jenkins/controllers/jjb/jenkins-jjb-controller.yaml

```diff
@@ -95,18 +113,22 @@ objects:
 - apiVersion: v1
   kind: DeploymentConfig
   metadata:
-    labels:
-      app: "${NAME}"
     name: "${NAME}"
+    labels:
+      app: "${APP}"
+      component: "${COMPONENT}"
   spec:
     replicas: 1
     selector:
```

Comment:
You don't need to set the selector and metadata labels when you have set pod template labels.

Reply:
Note that they are planning on breaking this assumption.

Reply:
those sons of guns